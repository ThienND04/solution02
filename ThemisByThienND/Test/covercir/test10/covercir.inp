5
924 403  247
965 492  97
297 10  61
649 596  58
797 623  446
