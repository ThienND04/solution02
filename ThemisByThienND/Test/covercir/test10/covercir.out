539
511
173
379
472
